docs/root/configuration/http/http_filters/_include/credential-injector-generic-filter.yaml

```diff
@@ -1,10 +1,6 @@
 static_resources:
   listeners:
   - name: listener_0
-    address:
-      socket_address:
-        address: 0.0.0.0
-        port_value: 10000
```

Comment:
trying to understand why this is removed

Reply:
thanks for catching. fat fingers. will revert.